static/panes/compiler.js

```diff
@@ -132,13 +132,19 @@ function Compiler(hub, container, state) {
         monacoDisassembly = languages[this.currentLangId].monacoDisassembly;
     }
 
-    this.outputEditor = monaco.editor.create(this.monacoPlaceholder[0], monacoConfig.extendConfig({
```

Comment:
Unfortunately like you said the auto format here has obscured the real change. Hoping if we reformat compiler.js in main and then you rebase, it'll revert to just being the diffs we'd expect.